storybook/stories/API/chart/ScatterChart.stories.tsx

```diff
@@ -1,9 +1,117 @@
 import React from 'react';
+import { Args } from '@storybook/react';
 import { coordinateData } from '../../data';
 import { ResponsiveContainer, Scatter, ScatterChart, XAxis, YAxis } from '../../../../src';
 
+const GeneralProps: Args = {
+  width: {
```

Comment:
As most of these are rather generically shared between stories, it might be nice to remove redundant definitions and share the descriptions.

Reply:
Not blocking though, lets iterate.